### PR TITLE
Take two alerts

### LINKED ISSUE #395: Alerts should always show below the header.

### LINKED ISSUE #454: Duplicate alert messages and positioning
Screen Shot 2020-10-01 at 9 39 06 AM

Screen Shot 2020-10-01 at 9 54 23 AM

### PR DESCRIPTION
**Pull request by mmazanec22**
Closes #454 
Closes #395 for real this time
Depends on https://github.com/ctoec/component-library/pull/75

### CODE REVIEW

**Review by jlhogan — APPROVED** [review]
the re-formatting of `client/package.json` is a mystery that will live to see another day

**Comment by mmazanec22** [comment]
@jlhogan I finally dealt with it bc it was making rebasing annoying on another branch.  Sorry for the accidental weeks long tabs vs spaces fight caused by an errant text editor setting!

File: client/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ctoec/component-library": "https://github.com/ctoec/component-library#base",
+    "@ctoec/component-library": "https://github.com/ctoec/component-library#multi-alert-spacing",
```
Comment (kmagida110) [inline client/package.json:6]: Process question here, does this get updated to base again when the new change merges or should it stay as base and we wait until this branch merges?

**Review by kmagida110 — APPROVED** [review]
Approved, one question for my own knowledge

File: client/package.json (hunk 1 above)
Comment (mmazanec22) [reply client/package.json:6]: I will need to manually update this branch to base again before merging this change

**Pull request merged**